docs/content/en/schemas/v2alpha3.json

```diff
@@ -494,6 +494,15 @@
           "description": "builder image used.",
           "x-intellij-html-description": "builder image used."
         },
+        "buildpacks": {
```

Comment:
I'm surprised this came up here. Can you rerun `hack/generate-schemas.sh`?

Reply:
Fixed, I did something wrong with the branch creating, sorry